src/DetailsView/Styles/detailsview.scss

```diff
@@ -888,16 +889,10 @@ div.insights-file-issue-details-dialog-container {
                 }
                 .issues-table {
                     height: 100%;
-                    width: 100%;
+                    width: calc(100% - 48px);
```

Comment:
Can this use a comment/var to indicate where the 48px is coming from?

Reply:
Sure; let me look into doing that.

Reply:
Done! 😄